Documentation generation failing on Github

Documentation is failing to build on GitHub. 

```
[ Info: SetupBuildDirectory: setting up build directory.
[ Info: Doctest: running doctests.
[ Info: ExpandTemplates: expanding markdown templates.
Error: Process completed with exit code 1.
```

Not much to go on there.

Now I get...

```
└ @ Documenter.Expanders ~/.julia/packages/Documenter/pjwqp/src/Expanders.jl:36
┌ Debug: Running ExpanderPipeline on api.md
└ @ Documenter.Expanders ~/.julia/packages/Documenter/pjwqp/src/Expanders.jl:39
┌ Debug: Running ExpanderPipeline on index.md
└ @ Documenter.Expanders ~/.julia/packages/Documenter/pjwqp/src/Expanders.jl:39
┌ Debug: Running ExpanderPipeline on partitionDS.md
└ @ Documenter.Expanders ~/.julia/packages/Documenter/pjwqp/src/Expanders.jl:39
┌ Debug: Running ExpanderPipeline on shist.md
└ @ Documenter.Expanders ~/.julia/packages/Documenter/pjwqp/src/Expanders.jl:39
┌ Debug: Running ExpanderPipeline on stopwatch.md
└ @ Documenter.Expanders ~/.julia/packages/Documenter/pjwqp/src/Expanders.jl:39
Error: Process completed with exit code 1.
```

--- a/docs/make.jl
+++ b/docs/make.jl
@@ -1,7 +1,5 @@
 using Documenter, IRMA
 
-ENV["JULIA_DEBUG"] = Documenter
-
 makedocs(
     sitename = "IRMA Docs",
     format = Documenter.HTML(),

--- a/docs/src/stopwatch.md
+++ b/docs/src/stopwatch.md
@@ -2,7 +2,12 @@
 
 It is often important to time operations in MPI. The `StopWatch` object will do that for you by recording the current MPI time (with `MPI.Wtime`) when you ask for a "stamp" given a label. The "start" stamp is recorded automatically when the StopWatch is created. You add more stamps with the `stamp` function. If you want to transport the StopWatch over MPI, then you can turn it into a NamedTuple with `asNamedTuple` (the NamedTuple will be `isbits` compatible).
 
-```@repl
+```julia
+# Note that this block is not run because it crashes the GitHub Action
+# that generates the documentation. I think this happens because
+# MPI.Wtime is called.  So this sketch of what to do will have
+# to suffice
+
 using IRMA
 
 sw = StopWatch()  # The start stamp is automatically recorded

